apps/multisig/src/layouts/Overview/Transactions/TransactionsList.tsx

```diff
@@ -160,18 +160,16 @@ export const TransactionsList = ({
                               }
                             : undefined,
                         }}
-                        onApproved={({ result, executed }) => {
-                          if (executed) {
-                            setUnknownTransactions(prev => [
-                              ...prev,
-                              `${multisig.id}-${makeTransactionID(
-                                multisig.chain,
-                                result.blockNumber?.toNumber() ?? 0,
-                                result.txIndex ?? 0
-                              )}`,
-                            ])
+                        onApproved={() => {
+                          if (openTransaction?.decoded?.changeConfigDetails) {
+                            updateMultisigConfig({
+                              ...multisig,
+                              threshold: openTransaction.decoded.changeConfigDetails.threshold,
+                              signers: openTransaction.decoded.changeConfigDetails.signers,
+                            })
```

Comment:
this should only execute if the transaction is executed, otherwise  the config will be updated before the change is executed on chain